src/components/Game.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import BeforeGame from './BeforeGame';
+
+
+class Game extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onStartGame = this.onStartGame.bind(this);
+
+        this.state = {
+            currentGameStatus: 'before',
```

Comment:
we should create a new file `gameStates` kept in `src/constants` folder to have easier life in future and import it here.

Reply:
I honestly believe that it would be nice if we do it together lataer, is it alright gentleman ?